SimG4CMS/Calo/interface/CaloSD.h

```diff
@@ -84,6 +84,9 @@ class CaloSD : public SensitiveCaloDetector,
   void resetForNewPrimary(const G4Step*);
   double getAttenuation(const G4Step* aStep, double birk1, double birk2, double birk3) const;
 
+  const std::string printableDecayChain(std::vector<unsigned int> decayChain);
```

Comment:
Based on the implementation this should be
```suggestion
  static std::string printableDecayChain(const std::vector<unsigned int>& decayChain);
```
(`const` on return value is meaningless, and in fact somewhat harmful by preventing RVO). I'm suggesting to make it `static` because the function does not use any member variables, making it `const` could also work.